src/components/ebay-combobox/combobox.stories.js

```diff
@@ -48,7 +48,7 @@ export default {
             description:
                 "default is `none`; available values are `none` or `list`. For list, will automatically filter results while typing.",
         },
-        "list-selection": {
+        listSelection: {
```

Comment:
I'm not sure why changing the name actually makes this work since there are other options that have the same naming convention. Maybe some internals were not aligned with the name?

Reply:
Even I was surprised with that. I see at some places where this works just fine. may be @agliga can shed some light.